scripts/checks/inheritance-ordering.js

```diff
@@ -13,7 +13,7 @@ for (const artifact of artifacts) {
   const linearized = [];
 
   for (const source in solcOutput.contracts) {
-    if (source.includes('/mocks/')) {
+    if (['contracts-exposed', 'contracts/mocks'].some(pattern => source.match(pattern))) {
```

Comment:
Not a big deal but keeping the slashes and using `startsWith` makes the pattern more robust. `match` will cast the argument to a regex so better to avoid unless it's an actual regex.

```suggestion
    if (['contracts-exposed/', 'contracts/mocks/'].some(pattern => source.startsWith(pattern))) {
```